content/pillow/concepts/image/terms/resize/resize.md

```diff
@@ -0,0 +1,123 @@
+---
+Title: '.resize()'
+Description: 'The .resize() method is part of the Image class in the Pillow library (PIL Fork) in Python, and it is used to change the size of an image.'
+Subjects:
+  - 'Data Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Computer Vision'
+  - 'Images'
+  - 'Pillow'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science-foundations'
+---
+
+The **.resize()** method is part of the **Pillow** library in Python, and it's used to change the size of an image. Pillow supports image formats such as JPEG, PNG, GIF, TIFF, WebP, and can be installed on a local PC using **pip**.
+
+## Syntax
+
+```pseudo
+Image.resize(size, resample=None, box=None, reducing_gap=None)
+```
+
+| Parameter Name | Data Type   | Usage                                                     |
+| -------------- | ----------- | --------------------------------------------------------- |
+| `size`         | tuple       | Decides the resize dimentions.                            |
+| `resample`     | int         | Specifies the resampling filter to be used when resizing. |
+| `box`          | tuple, None | A 4-tuple defining the region of the image to resize.     |
+| `reducing_gap` | float, None | Used for large scale reduction to help preserve quality.  |
+
+- resample filters available
+  - Image.Resampling.NEAREST or Image.NEAREST
+  - Image.Resampling.BOX
+  - Image.Resampling.BILINEAR or Image.BILINEAR
+  - Image.Resampling.HAMMING
+  - Image.Resampling.BICUBIC or Image.BICUBIC
+  - Image.Resampling.LANCZOS or Image.LANCZOS
+
+Returns an **image** **object**.
```

Comment:
For the parameters description, please use the following format:

- `size`: Parameter description
- `resample`: Parameter description
- `box`: Parameter description
- `reducing_gap`: Parameter description

You can avoid giving the details of the resample filters for this entry. Only the parameters description should be given.